Gemfile.lock

```diff
@@ -256,8 +281,14 @@ GEM
     zinzout (0.1.1)
 
 PLATFORMS
```

Comment:
Not sure how we ended up with so many additional platforms here? 🤷

Reply:
Dunno. Replacing `Gemfile.lock` with a fresh copy from `mariadb` branch and doing the minimal required changes cleaned it up.